.github/workflows/stale.yml

```
@@ -21,21 +21,20 @@ jobs:
     steps:
       - uses: actions/stale@v6
         with:
-          debug-only: false
+          debug-only: true
           days-before-issue-stale: ${{ env.BEFORE_ISSUE_STALE }}
           days-before-issue-close: ${{ env.BEFORE_ISSUE_CLOSE }}
           stale-issue-message: |
             This issue has been marked as stale due to inactivity for the last ${{ env.BEFORE_ISSUE_STALE }} days.
             It will be automatically closed in ${{ env.BEFORE_ISSUE_CLOSE }} days.
           close-issue-message: |
-            Hi everyone! This issue has been closed due to inactivity.
+            Hi everyone! This issue has been automatically closed due to inactivity.
             If you think this issue is still relevant in the latest Solidity version and you have something to [contribute](https://docs.soliditylang.org/en/latest/contributing.html), feel free to reopen.
             However, unless the issue is a concrete proposal that can be implemented, we recommend starting a language discussion on the [forum](https://forum.soliditylang.org) instead.
-          any-of-issue-labels: stale # TODO: remove this when we're done with closing ancient issues
-          ascending: true # TODO: remove this when we're done with closing ancient issues
+          ascending: true
           stale-issue-label: stale
           close-issue-label: closed-due-inactivity
-          exempt-issue-labels: 'bug :bug:,roadmap,selected-for-development,must have'
+          exempt-issue-labels: 'bug :bug:,epic,roadmap,selected-for-development,must have,must have eventually,SMT'
```

```suggestion
          exempt-issue-labels: 'bug :bug:,roadmap,selected-for-development,must have,must have eventually,SMT'
```

According to https://github.com/ethereum/solidity/pull/13947#issuecomment-1425573394, we don't want `epic` here after all.

Also, it's not clear from that comment what @NunoFilipeSantos's opinion is on exempting `should have` and `nice to have`. I'm fine with going either way, but I just want to make sure it's configured the way he meant it.

---

It is, I already submitted the code suggestions. Let's stop stalling and merge this PR. There's nothing left to be discussed.

---

I don't think it's fair to call me out for stalling here. I specifically came here on the weekend to approve this to make sure did not wait until Monday. And the only reason I did not merge the PR was that I noticed things that seemed to not match what we agreed on, even though I personally think we would have been better off not changing them.

---

Ah sorry, I should have made this comment on the issue instead it went into this thread, it was not directed at you. 🤔 
I blame GitHub mobile.